docs/indicator-configuration.md

```diff
@@ -298,6 +311,17 @@ graph_limits:
 graph_stacked_disaggregation: Age
 ```
 
+### graph_target_lines
+
+**_Optional_**: This setting can be used to add "target line" annotations to the graph. Like `graph_titles` it can include multiple items, and each can be limited to particular units or series. Each item can have the following settings:
+
+    * `series`: If specified, the target line will only display when the user is looking at this series.
+    * `unit`: If specified, the target line will only display when the user is looking at this unit of measurement.
+    * `label`: The text to display on the annotation. Defaults to "2030 target".
+    * `value`: The value at which to draw the line. This number corresponds to your actual data.
+
+Note that this setting relies on the Chart.js annotation plugin. If you would like to tweak any other settings for more site-wide control, you can override the `_includes/components/annotation_presets.js` file.
```

Comment:
Same as above - should this be `_includes/components/charts/annotation_presets.js`?